### DISCUSSION TITLE
Best Compatible MVVM toolkit?

### DISCUSSION THREAD

**Discussion by AmeerMansour** [MahApps/MahApps.Metro#4372] [Q&A]
there are a lot of toolkits of MVVM , which will be best to use with metro and material design?

**Comment by timunie** [comment]
Honestly, I think this is not related to MahApps in any way. MahApps will provide the components of the UI, but the MVVM Framework will provide the underlaying ViewModel. 

I personally like the Community MVVM Toolkit. If I have more complex Apps I use Reative UI, which also can be combined with the Community MVVM Toolkit. Or you can even write your own for prototyping a smaller App. 

Happy coding
Tim
